tests/build/jvm-build.go

```diff
@@ -399,7 +399,7 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jv
 
 				imageWithDigest := i.Spec.Image + "@" + i.Spec.Digest
 
-				Expect(f.AsKubeAdmin.TektonController.AwaitAttestationAndSignature(imageWithDigest, 5*time.Minute)).To(
+				Expect(f.AsKubeAdmin.TektonController.AwaitAttestationAndSignature(imageWithDigest, 30*time.Minute)).To(
```

Comment:
nitpick: will it be good to create a common variable [here](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/const.go) and use the same in both `build_templates.go` and `jvm-build.go`

Reply:
Yup, I can do that

Reply:
Hmm, maybe it should be in `pkg/constants/constants.go` instead and be used for the enterprise-contract tests as well?

Reply:
Added a common constant :+1: